tensorflow/compiler/mlir/tensorflow/tests/tf_interface_optimize_pass.mlir

```diff
@@ -0,0 +1,136 @@
+// RUN: tf-opt %s -tf-interface-optimize-pass | FileCheck %s
+
+// CHECK-LABEL: testSingleConj
+func @testSingleConj(%arg0: tensor<8x16x32x64xcomplex<f32>>) -> tensor<8x16x32x64xcomplex<f32>> {
```

Comment:
nit: I believe this test can be simplified by using a scalar tensor `tensor<complex<f32>>`